torch/lib/c10d/ProcessGroupNCCL.hpp

```diff
@@ -703,6 +727,6 @@ class ProcessGroupNCCL : public ProcessGroup {
   // by 1 when ncclGroupStart() is called and decreased by 1 when ncclGroupEnd()
   // is called.
   static thread_local uint64_t ncclActiveGroupCounter_;
-};
+}; // namespace c10d
```

Comment:
Did this come from clang-format? Doesn't seem right since the namespace actually ends below on line 732?

Reply:
I guess I somehow messed it up when resolving some merge conflicts.